### PR TITLE
task: add InsAVE-80K composed video retrieval

### LINKED ISSUE #5146: Add dataset: InsAVE-80K
### dataset link on Hugging dataset

https://huggingface.co/datasets/suimu/InsAVE-80K

### Arxiv link

https://arxiv.org/abs/2605.18467

### Description of the dataset

InsAVE-80K is a dataset introduced with InstructAV2AV for instruction-conditioned audio-visual/video retrieval and editing.

The relevant retrieval setting for MOEB is: video + text → video

A query combines a source video with a natural-language instruction or modification, and the goal is to retrieve the corresponding target video.

This dataset would provide additional coverage for instruction-conditioned composed video retrieval.

### PR DESCRIPTION
**Pull request by myang333**
Adds `InsAVE80KVT2VRetrieval` (`vt2v`, beta): **query = source video + editing instruction → target video**.

Closes #5146

### New MOEB construction, not an official benchmark
Derived from the official InstructAV2AV 1,000-pair **generation**-evaluation split. The paper (arXiv:2605.18467) scores generation quality there (FVD, FAD, PEAVS, Sync-C/D) and **defines no retrieval evaluation and no reference scores** — nothing here reproduces a published number.

### Construction
- **Queries**: the 1,000 official pairs — source clip + forward `instruction`, verbatim (incl. `<S>`/`<E>` markers).
- **Corpus**: all 2,000 released eval clips. Nothing added or dropped.
- **Source clips remain hard negatives**, deliberately not excluded, per `CIRRIT2IRetrieval`/`FashionIQIT2IRetrieval`. The instruction is what separates the edited clip from the unedited one.
- **Qrels**: the declared target plus any clip **byte-identical (SHA-256)** to it. The split has 60 add/remove reverse couples where one row's source *file* is the next row's target *file*, so the gold appears under two ids: 880 queries have 1 gold, 120 have 2 (1,120 total). Similarity never qualifies, only identical bytes.

### Data
Pinned revision `8ba8ccaad1d8f97b08e218dae1f2b439bd3d2289`. Only `eval.csv` and the single 1.55 GB eval shard are downloaded; the 139 GB training data is never touched.

### KNOWN_ISSUES
The `duplicate_video` entry records an intrinsic property of the release: the 2,000 files hold 1,872 unique video contents. Not masked — the qrel rule means no model is penalised for returning identical bytes under the other id.

### License
`not specified`: the repo declares MIT, but clips derive from YouTube/MovieBench/Condensed Movies/VGGSound and the card disclaims underlying media rights, as for YouCook2/ActivityNetCaptions.

### Validation
`mteb/baseline-random-encoder`: ndcg@10 0.0029, R@1 0.0, R@1000 0.473 (≈ chance). Offline loader tests cover collision and schema guards.

### CODE REVIEW

File: tests/test_tasks/test_insave_vt2v_loader.py
Comment (Samoed) [inline tests/test_tasks/test_insave_vt2v_loader.py:1]: You shoudn't add tests for tasks loading

File: mteb/tasks/retrieval/eng/insave_vt2v_retrieval.py [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+  year = {2026},
+}
+""",
+        prompt={
+            "query": "Given a source video and an editing instruction, retrieve the video that results from applying the instruction to the source video."
+        },
+        is_beta=True,
+    )
+
+    def load_data(self, num_proc: int | None = None, **kwargs) -> None:
```
Comment (Samoed) [inline mteb/tasks/retrieval/eng/insave_vt2v_retrieval.py]: This seems complicated. Can you reupload this task with fixed format? You can do this with `task.push_dataset_to_hub`

File: mteb/tasks/retrieval/eng/insave_vt2v_retrieval.py [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+      which carry 2 gold ids; the remaining 880 carry 1. Visual or semantic similarity
+      never qualifies -- only an identical SHA-256 digest does -- so a source clip can
+      never become relevant merely by resembling the target.
+
+    The closest upstream precedent for content-identity handling is `GreekCivicsQA`
+    and `XQuADRetrieval`, which hash document content to derive ids and thereby
+    collapse duplicates into a single document. That approach is not used here because
+    the corpus is meant to preserve every released file as its own candidate; qrel
+    expansion encodes the same fact without dropping ids.
+    """
```
Comment (Samoed) [inline mteb/tasks/retrieval/eng/insave_vt2v_retrieval.py]:
```suggestion
```

**Review by Samoed — CHANGES_REQUESTED** [review]

**Review by Samoed — APPROVED** [review]

**Comment by myang333** [comment]
The failing check is a Windows/Python 3.10 test that also fails in unrelated PRs. I don’t have permission to rerun it, could someone help rerun the failed job?
